experimental/Schemes/src/IdealSheaves.jl

```diff
@@ -1200,6 +1201,10 @@ function Base.show(io::IO, I::AbsIdealSheaf)
 end
 
 function Base.show(io::IO, ::MIME"text/plain", I::PrimeIdealSheafFromChart)
+  if has_attribute(I, :name)
+    print(io, get_attribute(I, :name))
+    return
+  end
```

Comment:
I think we have the macro `@show_special` https://nemocas.github.io/AbstractAlgebra.jl/dev/misc/#AbstractAlgebra.PrettyPrinting.@show_special for this.